packages/perseus/src/widgets/radio/__tests__/radio.testdata.ts

```diff
@@ -205,16 +234,16 @@ export const choicesWithImages: PerseusRenderer = {
     },
 };
 
+export const choicesWithImagesItem: PerseusItem =
+    createRadioItem(choicesWithImages);
+
+export const choicesAndImagesItem: PerseusItem =
+    createRadioItem(choicesWithImages);
+
 export const multiChoiceQuestion: PerseusRenderer = {
     content:
         "**Select all input values for which $g(x)=2$.**\n\n[[\u2603 radio 1]]\n\n ![](web+graphie://ka-perseus-graphie.s3.amazonaws.com/4613e0d9c906b3053fb5523eed83d4f779fdf6bb)",
-    images: {
-        "web+graphie://ka-perseus-graphie.s3.amazonaws.com/4613e0d9c906b3053fb5523eed83d4f779fdf6bb":
-            {
-                width: 428,
-                height: 428,
-            },
-    },
```

Comment:
This content was removed because the image is already in the content section and having an image here seems to cause a style issue. This is referenced in the investigation ticket [LEMS-2948](https://khanacademy.atlassian.net/browse/LEMS-2948). Changing this caused an update to the snapshots as well.

Reply:
This style issue is unrelated to LEMS-2948, likely just a problem with styling in the fake Perseus Chrome we use in SB. Removing the image only sweeps the issue under the rug, we need a separate ticket to look into the styles causing this.

Reply:
Created a separate ticket here: https://khanacademy.atlassian.net/browse/LEMS-2955